### ISSUE TITLE
Getting CKEditor phpBB Plugin and Media Embed phpBB Plugin working together

### ISSUE THREAD

**Issue by stephanedaviet** [xchwarze/ckeditor-phpbb#2]
Great addon, much more convenient than a bare BBCode editor for end users.

I'm just trying to make it work with the [Media Embed phpBB Plugin](https://www.phpbb.com/customise/db/extension/mediaembed/). I'm a bit lost on how to achieve this.

First of all, when both _CKEditor phpBB Plugin_ and _Media Embed phpBB Plugin_ are enabled, the last one doesn't work. `[media]` tags are not interpreted and stay as is in the submitted post.

Then, I've tried to add a custom `[media]` BBCode, but it doesn't work either. When flagging it as to be displayed on posting, I got the button, but no more luck: it isn't transformed into an embed media. For this specific test, two points:
- It's not really astonishing that a custom `[media]` BBCode doesn't work as _Media Embed phpBB Plugin_ works a bit specially. It isn't registered as a custom BBCode and is based on event listener mechanism and works with s9e `TextFormatter`,
- Whatever the custom BBCode added in the CKEditor phpBB Plugin toolbar, it's the same: it's not converted to HTML and stay as vanilla BBCode in final message (tested with `[acronym]` and it doesn't work either).

Last, I also look the [Embed Plugin](https://ckeditor.com/cke4/addon/embed) and [Semantic Embed Plugin](https://ckeditor.com/cke4/addon/embedsemantic) for CKEditor, much more powerful than the _YouTube Plugin_ provided in _CKEditor phpBB Plugin_. Those plugins are based on the oEmbed convention and the [Iframely](https://iframely.com/plans) providing this service. It's quite conflicting with the principle of _Media Embed phpBB Plugin_ as this one is based on a list of websites configured inside phpBB itself.

TLDR; I don't know how to get those two plugins working together.

See also [Media embed plugin - CKEditor 4](https://www.phpbb.com/customise/db/extension/ckeditor_for_phpbb/support/topic/246046) asking for the same thing.

**Comment by xchwarze** [comment]
It would be necessary to debug some hours where the error is. As far as I remember (as I actually made this plugin more than 4 years ago and the phpbb validation was completed a few months ago....) the plugin I developed it as simple as possible so it does not affect any third party plugin.
My original idea was that this quickly pass the validation and with that feedback make a final and new plugin in ckeditor 5 but now 4 years later and I do not have that time.

So I do not really have space to really debug this case, but if you find how to fix it I can integrate those changes to this repo.
